tests/test_api.py

```diff
@@ -3659,3 +3660,357 @@ def test_operations(self):
         op8_args = json.loads(op8.args)
         self.assertEqual(len(op8_args), 1)
         self.assertEqual(op8_args['uidentity'], 'e8284285566fdc1f41c8a22bb84a295fc3c4cbb3')
+
+
+class TestUnmergeIdentities(TestCase):
+    """Unit tests for unmerge_identities"""
+
+    def setUp(self):
+        """Load initial dataset"""
+
+        self.user = get_user_model().objects.create(username='test')
+        self.ctx = SortingHatContext(self.user)
+
+        api.add_organization(self.ctx, 'Example')
+        api.add_organization(self.ctx, 'Bitergia')
+
+        Country.objects.create(code='US',
+                               name='United States of America',
+                               alpha3='USA')
+
+        api.add_identity(self.ctx, 'scm', email='jsmith@example')
+        api.add_identity(self.ctx,
+                         'git',
+                         email='jsmith-git@example',
+                         uuid='e8284285566fdc1f41c8a22bb84a295fc3c4cbb3')
+        api.update_profile(self.ctx,
+                           uuid='e8284285566fdc1f41c8a22bb84a295fc3c4cbb3', name='J. Smith',
+                           email='jsmith@example', gender='male', gender_acc=75)
+        api.enroll(self.ctx,
+                   'e8284285566fdc1f41c8a22bb84a295fc3c4cbb3', 'Example',
+                   from_date=datetime.datetime(1900, 1, 1),
+                   to_date=datetime.datetime(2017, 6, 1))
+
+        api.add_identity(self.ctx, 'scm', email='jsmith@bitergia')
+        api.add_identity(self.ctx,
+                         'phabricator',
+                         email='jsmith-phab@bitergia',
+                         uuid='e8284285566fdc1f41c8a22bb84a295fc3c4cbb3')
+        api.update_profile(self.ctx,
+                           uuid='e8284285566fdc1f41c8a22bb84a295fc3c4cbb3', name='John Smith',
+                           email='jsmith@profile-email', is_bot=True, country_code='US')
```

Comment:
The profile was already edited before. What's the purpose of this? Why don't you do it at the same time?

Reply:
You are right, I removed the unnecessary calls to `update_profile`.